mozilla-release/toolkit/mozapps/extensions/internal/XPIProvider.jsm

```diff
@@ -7953,6 +7955,7 @@ Object.assign(SystemAddonInstallLocation.prototype, {
       logger.warn(`System add-on ${aAddon.id} isn't restartless.`);
       return false;
     }
+    */
```

Comment:
was asking people on IRC about that, but did get much. As we have only two system-addons we are able to test for unexpected situations